### DISCUSSION TITLE
mana/tec/DynamicDataSource

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#680] [Announcements]
# mana/tec/DynamicDataSource

简介 动态多数据源组件，在应用中使用多数据源，并且能够使用便利的方式进行切换 。 使用 依赖 配置 - dbName 表示配置中心中的命名 use57 表示十分启用 57 的数据库; 集成数据库连接池 集成 druid 配置 1. 如果参数都未配置，则保持原组件默认值。 2. 如果配置了全局参数，则每一个数据源都会继承对应参数。 3. 每一个数据源可以...

http://localhost:8080/mana/tec/DynamicDataSource.html

**Comment by kaiguoyuanshuai** [comment]
试试评论区
